### PR TITLE
Do the same on dev and on Travis

### PR DESCRIPTION
**Pull request by sbrunner**
fix deploy daily pip package

### CODE REVIEW

File: travis/deploy-daily-pypi-build.sh [hunk 1]
```diff
@@ -12,4 +9,4 @@ echo "username:${TRAVIS_USERNAME}" >> ~/.pypirc
 echo "password:${TRAVIS_PASSWORD}" >> ~/.pypirc
 
 # Deploy daily pypi build
-.venv/bin/python setup.py egg_info --tag-date --tag-build=dev sdist bdist_wheel upload -r pypi
+python setup.py egg_info --tag-date --tag-build=dev sdist bdist_wheel upload -r pypi
```
Comment (ger-benjamin) [inline travis/deploy-daily-pypi-build.sh]: Are you sure that the project is build and ready to deploy ? (we use the container's python, I think that should pass but I'm not sure.)

**Comment by ger-benjamin** [comment]
Fro the daily, I close this one: https://github.com/camptocamp/pyramid_oereb/pull/235 ?

File: travis/deploy-daily-pypi-build.sh (hunk 1 above)
Comment (sbrunner) [reply travis/deploy-daily-pypi-build.sh]: Yes, I tested by changeing the deploy configuration :-)

**Comment by sbrunner** [comment]
Yes, thanks :-)

**Pull request merged**